pyproject.toml

```
@@ -35,6 +35,7 @@ chardet = ">=4,<6"
 validators = "0.20.0"
 gitpython = "^3.1.31"
 charset-normalizer = "^3.2.0"
+tomli = "^2.0.1"
```

```suggestion
tomli = { version = "^2.0.1", python = "<3.11" }
```
Python 3.11+ comes with a TOML parser included, so we could do this and then something like
```
if (version_info.major, version_info.minor) >= (3, 11):
   import tomllib
    tomllib.load(...)
else:
   import tomli
   tomli.load(...)
```

---

implemented in ec62dae9430946e354cdc76391d47e0917d8dea2.
But I use a slightly different structure because otherwise my lsp complains that the else part is unreachable.